MAINTAINERS.md

```diff
@@ -0,0 +1,199 @@
+
+If you can, close the issue with a comment that points the poster to the correct repo.
+
+#### `Need more info` for incomplete issues
+
+These are issues that seem worthwhile, but are not yet actionable. Give it the `Need more info` label and prod the author for that info.
+
+#### `Stale` for abandoned issues
+
+These are usually `Need more info` issues where the original author does not respond within a sensible time frame (say, a month. People are busy). These are good candidates for a Github bot to manage for you.
+
+Here’s [an example](https://github.com/openpgpjs/openpgpjs/issues/883) of an issue that was manually closed after the author not responding for a year.
+
+#### `Bug` for things that don’t work as intended
+
+1. Assuming this is not or no longer a `Need more info` issue, attempt to reproduce the bug
+   1. Search for duplicates, if one exists, point the current issue there, and close the new one
+   2. If you can reproduce it and there are no duplicates, prioritize it and add the `Bug` label
+      1. If you need more info before this issue becomes actionable despite being able to reproduce it, prod the author and add a `Need more info` label
+   3. If you can’t reproduce it, contact the author with the reason, and add a `Need more info` to the `Bug` label
+      1. If the issue cannot be reproduced by anyone other than the author, agree with them that the issue should be closed
+
+As with regular `Need more info` issues, `Need more info` + `Bug` issues are also subject to the [Abandoned/Stale](#abandonedstale) rule above, and should be closed after a set period of time.
+
+#### Good First Issues and Help Wanted Issues
+
+To identify issues that are specifically groomed for new contributors, we use the `Help wanted` _and_ `Good first issue` labels. As to the difference between the two:
+
+- a `Good first issue` always also implies `Help wanted` from outside the project
+- but `Help wanted` does not necessarily mean it’s a `Good first issue`. Might be really tricky but none of the current maintainers has the resources to tackle it
+
+##### Criteria for a `Help wanted` Issue
+
+- **Low Barrier to Entry:** It should be easy for new contributors. Documentation on how that type of change should be made should already exist.
+
+- **Clear Task:** The task is agreed upon and does not require further discussions in the community. Call out if that area of code is untested and requires new fixtures.
+
+- **Goldilocks priority:** The priority should not be so high that a core contributor should do it, but not too low that it isn't useful enough for a core contributor to spend time reviewing it, answering questions, helping get it into a release, etc.
+
+- **Up-To-Date:** Often these issues become obsolete and have already been completed, are no longer desired, no longer make sense, or have changed priority or difficulty.
```

Comment:
This seems like a very pessimistic comment in this context. Obviously it might be true but it's not really an actionable comment; obviously at the time when it's labeled as such it will be up-to-date, so I would just remove this tbh.

And again, IMO the main point of this label should be that the core maintainers won't have time for it and would welcome external contributors, so perhaps I would just only say that?